### DISCUSSION TITLE
Just tanks!

### DISCUSSION THREAD

**Discussion by WSTxda** [nyas1/Material-You-app-list#52] [General]
I always come here to see what's new!

This repository is simply necessary. 😍

**Comment by Maha-Rajan** [comment]
Could you add [Next player ](https://github.com/anilbeesetti/nextplayer) to the list. It's a video player with MDY support

**Reply by nyas1** [reply]
> Could you add [Next player ](https://github.com/anilbeesetti/nextplayer) to the list. It's a video player with MDY support

Sure. I have it installed and its awesome. I was just about to add to list 😅

**Comment by nyas1** [comment]
> I always come here to see what's new!
> 
> This repository is simply necessary. 😍

Exact same thoughts for your channel. Thank You.

**Comment by fallsdevil** [comment]
@nyas1 hey, could you add this markdown editor: [simple markdown ](https://github.com/wbrawner/SimpleMarkdown)
